bundles/core/src/test/resources/tabs/test-content.json

```diff
@@ -17,19 +17,140 @@
                         "jcr:title"         : "Tabs 1",
                         "sling:resourceType": "core/wcm/components/tabs/v1/tabs",
                         "activeItem"        : "item_2",
-                        "item_1": {
+                        "item_1"          : {
                             "jcr:primaryType"   : "nt:unstructured",
-                            "jcr:title"         : "Teaser 1",
-                            "jcr:description"   : "Teaser 1 description",
-                            "sling:resourceType": "core/wcm/components/teaser/v1/teaser"
+                            "jcr:title"         : "Tabs 1",
+                            "jcr:description"   : "Tabs 1 description",
+                            "sling:resourceType": "core/wcm/components/responsivegrid",
+                            "item_1": {
```

Comment:
I would rename all second level items as `item_x_y` and all third level items as `item_x_y_z`

Reply:
I still think we are missing a level here as we have `tabs-1/item_1/item_1` instead of `tabs-1/item_1/item_1_1`